tests/test_assessment_mapping_fund_round.py

```diff
@@ -12,27 +12,48 @@ def test_transform_to_assessor_task_list_metadata():
         {
             "name": "Organisation information",
             "sub_criterias": [
-                {"name": "Organisation information", "id": "org_info"},
-                {"name": "Applicant information", "id": "applicant_info"},
-            ],
+                {
+                    "name": "Organisation information",
+                    "id": "org_info",
+
```

Comment:
Cam we remove these whits spaces?

Reply:
https://github.com/communitiesuk/funding-service-design-assessment-store/pull/66/commits/3efc64b9d1731e4935e0c105e4780d67d4187700